pom.xml

```diff
@@ -179,7 +179,7 @@
     <jpam.version>1.1</jpam.version>
     <selenium.version>2.52.0</selenium.version>
     <paranamer.version>2.6</paranamer.version>
-    <maven-antrun.version>1.8</maven-antrun.version>
+    <selenium.htmlunitdriver.version>2.27</selenium.htmlunitdriver.version>    <maven-antrun.version>1.8</maven-antrun.version>
```

Comment:
@highfei2011 @srowen Reverted all pom.xml change since they appear not to be mandatory for this PR
Indeed it seems that selenium version should have been updated as well.